release-note-script/src/main/java/MergeReleaseNotes.java

```diff
@@ -185,10 +185,10 @@ public String getEdition() {
   }
 
   enum Category {
+    BACKWARD_INCOMPATIBLE("Backward incompatibles"),
```

Comment:
I'll change the display name from `Backward incompatibles` to `Backward-incompatible changes` since I got a request from @josh-wong

Reply:
Done. at dabd278